cosmosdb/sql/ps-container-create.ps1

```diff
@@ -13,7 +13,7 @@ $ContainerProperties = @{
             "kind"="Hash"
         }
     }; 
-    "options"=@{ "Throughput"="400" }
+    "options"=@{ "Throughput"= 400 }
 } 
 New-AzResource -ResourceType "Microsoft.DocumentDb/databaseAccounts/apis/databases/containers" `
     -ApiVersion "2015-04-08" -ResourceGroupName $resourceGroupName `
```

Comment:
I guess the above 4 files will be deleted?

Reply:
I will keep a couple of these container-level scripts moving forward (will be fully self-contained) as they are common scenarios and are not highly intuitive initially to write. If I receive feedback or data suggesting some of the container-level scripts are not being used I will revisit.

Reply:
yes, they will be deleted.